trk/src/api/TapsApi.js

```diff
@@ -123,7 +126,30 @@ function TapsApi() {
             .where("expiryTime", ">", firebase.firestore.Timestamp.now())
             .orderBy("expiryTime", "desc").get();
     }
-    
+
+    //To get the starting points of the last 5 expired sessions
+    function getRecentFiveSessions(userId) {
+        console.log('User ID: ', userId);
+        console.log('Fetching 5 Session Start Climbs....');
+        return ref
+            .where('user', '==', userId)
+            .where('isSessionStart', '==', true)
+            .where("expiryTime", '<=', firebase.firestore.Timestamp.now())
+            .orderBy("expiryTime", "desc")
+            .limit(5)
+            .get();
+    }
+
+    //To get all expired taps to build Expired sessions
+    function getExpiredTaps(userId) {
+        console.log('User ID: ', userId);
+        console.log('Fetching Expired Climbs....');
+        return ref
+            .where('user', '==', userId)
+            .where("expiryTime", '<=', firebase.firestore.Timestamp.now())
+            .orderBy("expiryTime", "desc")
+            .get();
+    }
```

Comment:
looks good, the .limit is to reduce loading time in profile right?

Reply:
Yupp, in the next PR ill make this dynamic on scrolling, will use the StartAfter mechanic for that with the same limit.